frontend/src/Context/WebSocketProvider.tsx

```diff
@@ -38,6 +71,21 @@ export function WebSocketProvider({ children }: { children: ReactNode }) {
         );
       };
 
+      ws.onmessage = event => {
+        console.log("Server is sending us message", event.data);
+
+        try {
+          const data = JSON.parse(event.data);
+
+          if (data.type === "notification") {
+            console.log("Got new notification", data.data);
```

Comment:
This should not be on prod, if is a temporary debug add a todo task here. Same for any console.log on the frontend